myapp/app/controllers/task_controller.rb

```diff
@@ -1,6 +1,12 @@
 class TaskController < ApplicationController
   def index
-    @tasks = Task.all
+    if params[:sort] == 'latest'
+      @tasks = Task.all.order(created_at: :desc)
```

Comment:
```suggestion
      @tasks = Task.order(created_at: :desc)
```
`all` is unnecessary, I guess?